docs/build_process.rst

```diff
@@ -3,6 +3,148 @@
+    * Verify existence and validity of parent images
+    * Verify that parent images come from existing koji build
+    * Resolve composes, integration with `odcs`_. See :ref:`yum-repositories-odcs-compose`
+      for details.
+    * For base image builds, create koji image task ("`image-build`_" task to create filesystems
+      for all required platforms) and include filesystem to build directory
+    * Supply "release" label if it is missing from Dockerfile and also reserve koji build nvr
+      in koji
+    * Apply automatic and configurable labels in Dockerfile
+    * Resolve and fetch remote sources if any requested
+    * For operator bundle builds, pin operator digests
+    * Generate and add help if required
+    * Fetch artifacts from PNC or URLs if requested
+    * Generate and add image content manifest
+    * Include yum repos, either from explicitly provided repos, or from odcs composes
+
+- Build tasks:
+    * Run multiple build tasks based on requested platforms to build for and push images
+      in the registry
+
+- Postbuild task:
+    * For flatpak builds, create OCI image, and push it to registry
+    * Query platform specific images to discover installed RPM packages, used later for compare
+      packages and koji metadata
+    * For operator bundle builds, fetch and zip operator manifest, used later for importing
+      to koji build
+    * Compare packages installed in images for all platforms
+    * Create a `manifest list`_ object in the registry, grouping together the image manifests
+      for all platforms
+    * Push floating tags to the registry for a manifest list
+    * Import the Koji build
+    * Tag the Koji build
+
+- Set results task:
+    * Set pipeline run results based on outputs of previous tasks
+
+- Exit task:
+    * For failed builds, cancel koji build nvr reservation
+    * Set annotation with build metadata in pipeline run
+    * Send email notifications if required (with default configuration for failed builds)
+
+.. _`image-build`: https://docs.pagure.org/koji/image_build/
+.. _`manifest list`: https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list
+.. _`odcs`: https://pagure.io/odcs
+
+Source pipeline consists of these tasks:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Build task:
+    * Fetch all sources from binary build (srpms, remote sources, artifacts)
+    * Reserve koji build nvr in koji
+    * Build source image using `BuildSourceImage`_
+    * Tag and push source image to the registry
+    * Import the Koji build
+    * Tag the Koji build
+
+- Set results task:
+    * Set pipeline run results based on outputs of previous tasks
+
+- Exit task:
+    * For failed builds, cancel koji build nvr reservation
+    * Set annotation with build metadata in pipeline run
+
+.. _`BuildSourceImage`: https://github.com/containers/BuildSourceImage
+
+Binary workflow
+---------------
+
+.. image:: images/osbs2_binary.png
+   :alt: OSBS binary detail workflow diagram
```

Comment:
Regardless of how much I zoom in my browser, these are still too small to read :-/

Reply:
you have to click on the image to open it, I can't do much about it .....

it is there if you really want to see it I can't make it visible on such small area